cloudbuild.yaml

```diff
@@ -36,16 +58,17 @@ steps:
 timeout: 1800s
 images:
   - ${_IMAGE_NAME}:${_IMAGE_VERSION}
+  - ${_DEPLOY_IMAGE_NAME}:${_IMAGE_VERSION}
```

Comment:
images defined here will be automatically pushed to registry, so no need to additionally use `docker push`